aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -242,6 +247,8 @@ private void assertRemoteSubscriptionsCancelled() {
     }
 
     private void assertRemoteSubscriptionsStarted() {
+        // Introduce a delay to allow the orchestrator to start since this is done in a non-blocking manner now.
+        Single.just(true).delay(ASSERTION_TIMEOUT_MS, TimeUnit.MILLISECONDS).blockingGet();
```

Comment:
We should keep this event driven, even from test. There is an initialization event that is published, currently. The intent of that event was to prevent the need for waits, in test. Between it, and the `HubAccumulator`, you should be able to await the initialization results. It's a more determinist mechanism than tuning `ASSERTION_TIMEOUT_MS`.

Reply:
That makes sense. I just changed it so the orchestrator emits a hub event on start and stop. Then modified the test to use the HubAccumulator. It will be in my next commit.